components/texts/text-list.css

```diff
@@ -0,0 +1,59 @@
+.texts {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.heading, .tx, .gr-tx {
+    margin: 0.5rem;
+}
+.medium-text {
+    font-size: 20px;
+}
+.gr-tx {
+    color: darkgray;
+}
+
+.center-text {
+    width: 500px;
```

Comment:
can you change it to rems?

Reply:
Made the changes you suggested, please review it again